src/components/cc-env-var-form/cc-env-var-form.stories.js

```diff
@@ -153,9 +153,12 @@ export const saving = makeStory(conf, {
     {
       appName: 'Foobar backend python',
       context: 'env-var',
-      state: { type: 'saving', validationMode: 'simple', variables: VARIABLES_FULL },
+      state: { type: 'loaded', validationMode: 'simple', variables: VARIABLES_FULL },
     },
   ],
+  onUpdateComplete: (component) => {
```

Comment:
**nitpick:** let's add a small comment explaining why we did that so that nobody tries to remove it :sweat_smile: 
```suggestion
  // the component needs to be loaded with data for it to display this data in saving mode afterward
  onUpdateComplete: (component) => {
```

(don't hesitate to improve the text :wink:)